sdk/storage/Azure.Storage.Queues/README.md

```diff
@@ -125,6 +125,63 @@ QueueClient queue = new QueueClient(accountUri, new DefaultAzureCredential());
 
 Learn more about enabling Azure Active Directory for authentication with Azure Storage in [our documentation][storage_ad] and [our samples](#next-steps).
 
+### Message encoding
+
+This version of library does not encode message by default. V11 and prior versions base64-encoded messages by default.
+This section shows how message encoding can be customized as well as additional error handling can be provided for poison messages.
```

Comment:
Let's also say Azure Functions uses base64 encoded messages by default so you should consider using this feature if you're integrating with a function.

Reply:
done.